### DISCUSSION TITLE
zh_cn.json(基本汉化+风格汉化完整,基本能看懂了)

### DISCUSSION THREAD

**Discussion by chouhai2018** [lllyasviel/Fooocus#2864] [Show and tell]
[zh_cn.json](https://github.com/lllyasviel/Fooocus/files/15208106/zh_cn.json)
不会写,下载后放到Fooocus\language目录里面
.bat那个批处理文件,加个--language zh_cn的参数,用这个批处理启动就行了
chatgpt汉化的,有些英文原版没有,我不会程序,也不会找,关键是把风格汉化全了,能看懂,谁程序好,可以再找找

**Comment by mashb1t** [comment]
@chouhai2018 thanks for the translation. Could you merge your changes with https://github.com/lllyasviel/Fooocus/pull/756 or https://github.com/lllyasviel/Fooocus/discussions/2494?

**Reply by chouhai2018** [reply]
这个合并,我不知道如何做啊,我非专业程序开发者
